packages/app/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Miscellaneous
```

Comment:
Isn't the global .gitignore enouph

Reply:
Thanks for the review Marvin. 

For the "desktop" folders, there is an issue about the support of those platforms.
We won't release on them, but it will allow us to run the app without an emulator/simulator.

For the .gitignore, we have the same file in the `smooth_app` package. It's actually the generated one.